package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
this file should be inside the /cypress directory to separate outer python backend from playwright's and cypress' dependencies